content/courses/program-security/bump-seed-canonicalization.md

```diff
@@ -151,17 +121,17 @@ pub fn set_value_secure(
 }
 ```
 
-### Use Anchor’s `seeds` and `bump` constraints
+### Use Anchor's `seeds` and `bump` constraints
 
 Anchor provides a convenient way to derive PDAs in the account validation struct
 using the `seeds` and `bump` constraints. These can even be combined with the
 `init` constraint to initialize the account at the intended address. To protect
-the program from the vulnerability we’ve been discussing throughout this lesson,
+the program from the vulnerability we've been discussing throughout this lesson,
 Anchor does not even allow you to initialize an account at a PDA using anything
 but the canonical bump. Instead, it uses `find_program_address` to derive the
 PDA and subsequently performs the initialization.
 
-```rust
+```
```

Comment:
Why are you removing syntax highlighting here? Please restore it unless there's a very good reason.

Reply:
Apologies had forgot to add them back in